core/embed/rust/src/protobuf/obj.rs

```diff
@@ -67,7 +67,7 @@ impl MsgObj {
         match attr {
             Qstr::MP_QSTR_MESSAGE_WIRE_TYPE => {
                 // Return the wire ID of this message def, or None if not set.
-                Ok(self.msg_wire_id.map_or(Obj::const_none(), |wire_id| wire_id.into()))
+                Ok(self.msg_wire_id.map_or_else(Obj::const_none, Into::into))
```

Comment:
why do you prefer `map_or_else` here?

Reply:
So that `Obj::const_none` will get called only in the `None` case. In this case it's a matter of style, really, buy there's also a lint for this that would complain with more sensitive setup.